libs/domain/search/facet-navigation/facet-navigation.component.ts

```diff
@@ -23,7 +28,11 @@ import { searchFacetNavigationStyles } from './facet-navigation.styles';
   expandedItemsCount: 5,
   valueRenderLimit: 5,
   minForSearch: 13,
-  bury: [{ facets: ['rating', 'price'] }],
+  bury: [
+    {
+      facets: ['price'],
```

Comment:
actually it's a breaking change. We could revert it and let the rating facet compnoent deal with it